.eslintrc.js

```diff
@@ -1,7 +1,18 @@
 module.exports = {
-    extends: 'ls-react',
+    parserOptions: {
```

Comment:
Is there a specific reason the lint settings are being changed? I think that for the sake of trying to also get this merged into the upstream fork, then fewer changes will result in a better chance of getting it merged.

Reply:
The eslint config repo ([https://github.com/lawnstarter/eslint-config-ls-react](https://github.com/lawnstarter/eslint-config-ls-react) got removed/archived completely, so you can't build the project without changing this.

Maybe the author uses this as a private library, so i can exclude those changes from the upstream PR. Still, the upstream repo has not had any changes since 2 years, so i don't know how successful the upstream PR will be

Reply:
OK, got it. Thanks for explaining! It's probably OK then. We could also consider making this change in a separate PR to decrease the scope/review of each PR.

Reply:
Actually, now that I think about this a little more... I would like them split out into separate PRs. I think that will help maintaining the changes in this fork easier. Could you please create a separate PR with just the ESlint change, and then I'll merge that, then come back here and merge this one.

Reply:
Done! Added a PR here: #4

Reply:
All right, that's merged now if you want to go ahead and merge main back into this PR to clean up the diff.

Reply:
Oh, never mind remerging main, I see the diff is fine now.